### PR TITLE
Create player-feedback.md

### PR DESCRIPTION
**Pull request by LuniMoon**
1st overview of the structure for "Player Feedback" (I'll continue after lunch break)

### CODE REVIEW

File: docs/gdevelop5/interface/games-dashboard/player-feedback.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+## Habilitating Player Feedback
+Player feedback can be turned on or off acording to the game's development stage.
+
+To be able to collect player feedback the game has to be available online. You can verify that the Game has been published to gd.games through the [Manage section](https://wiki.gdevelop.io/gdevelop5/interface/games-dashboard/):
+
+![Game-Manager](Manage-Games.png)
+
+On the game listing screen you can configure:
+* **Make discoverable on gd.games:** players will be avaliable to find the game through the gd.games site
```
Comment (AlexandreSi) [inline docs/gdevelop5/interface/games-dashboard/player-feedback.md:34]: When you start a list, you must an empty line before the first element:
```suggestion
On the game listing screen you can configure:

* **Make discoverable on gd.games:** players will be avaliable to find the game through the gd.games site
```

File: docs/gdevelop5/interface/games-dashboard/player-feedback.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+* **Show feedback banner on gd.page:** the game will display a banner to collect player feedback through the site gd.games
+* **Ask for feedback on all builds:** All published versions of the game listed under the "Builds" tab will dislay the feedback banner to collect player feedback
+
+To learn how to publish to the gd.games platform and configure the privacy of the links [follow these instructions](https://wiki.gdevelop.io/gdevelop5/publishing/web/)
+
+## Recieving feedback
```
Comment (AlexandreSi) [inline docs/gdevelop5/interface/games-dashboard/player-feedback.md:40]:
```suggestion
## Receiving feedback
```

Comment (AlexandreSi) [reply docs/gdevelop5/interface/games-dashboard/player-feedback.md:40]: And below as well

File: docs/gdevelop5/interface/games-dashboard/player-feedback.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+Click on the [Manage section](https://wiki.gdevelop.io/gdevelop5/interface/games-dashboard/) > the game that you wish to manage to see the "Feedback tab".
+
+![Feedback-tab](Player-feedback.png)
+
+## Rating Feedback
+Rating player feedback helps the community identify the best beta testers of gd.games so they can get rewarded accordingly.
+
+![Rate player feedbal](Rate-feedback.png)
+
+Rate the feedback by clicking on the correspondent icon:
```
Comment (AlexandreSi) [inline docs/gdevelop5/interface/games-dashboard/player-feedback.md:51]: You must add a new line here as well (the mardown interpreter is very strict for the documentation)

File: docs/gdevelop5/interface/games-dashboard/player-feedback.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+It helps developers understand player interactions, preferences, and areas needing improvement in order to encrease game adoption by their final target market for game success.
+
+GDevelop's intergrated player feedback allows developers to enhance game design to increase player satisfaction, leading to a more engaging experience and a loyal player base.
+
+## Feedback availabity by channel
+Player Feedack is available to every user with a GDevelop account from free to paying subscriptions and *only* through gd.games.
+Games published on sites like Itch, Newgrounds, Poki, and personal website; or exported to Android or iOS cannot collect player feedback.
+
+Games published on gd.games have the possibility to be:
```
Comment (AlexandreSi) [inline docs/gdevelop5/interface/games-dashboard/player-feedback.md:20]: Here as well for the list

File: docs/gdevelop5/interface/games-dashboard/player-feedback.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+* Unlisted on gd.games only available to people with the URL
+* Uploaded to the gd.games server, but not available through the public site (recommended for private player feedback)
+
+
+## Habilitating Player Feedback
+Player feedback can be turned on or off acording to the game's development stage.
+
+To be able to collect player feedback the game has to be available online. You can verify that the Game has been published to gd.games through the [Manage section](https://wiki.gdevelop.io/gdevelop5/interface/games-dashboard/):
+
+![Game-Manager](Manage-Games.png)
```
Comment (AlexandreSi) [inline docs/gdevelop5/interface/games-dashboard/player-feedback.md:31]: This first image is really low quality. Could you add one with a better resolution?

Comment (AlexandreSi) [reply docs/gdevelop5/interface/games-dashboard/player-feedback.md:31]: Oh, and the part between [] is the part that screen readers will read to the users so you can set a description like "Screenshot of the Manage section of GDevelop's homepage"

Comment (LuniMoon) [reply docs/gdevelop5/interface/games-dashboard/player-feedback.md:31]: I am trying, but I cannot seem to replace it like I did before? Lemme try again.

Comment (LuniMoon) [reply docs/gdevelop5/interface/games-dashboard/player-feedback.md]: Done!

File: docs/gdevelop5/interface/games-dashboard/player-feedback.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+## Receiving feedback
+Recieved player feedback is stored and visible under the "Feedback" tab of the desired published Game.
+Click on the [Manage section](https://wiki.gdevelop.io/gdevelop5/interface/games-dashboard/) > the game that you wish to manage to see the "Feedback tab".
+
+![Feedback-tab](Player-feedback.png)
+
+## Rating Feedback
+Rating player feedback helps the community identify the best beta testers of gd.games so they can get rewarded accordingly.
+
+![Rate player feedbal](Rate-feedback.png)
```
Comment (4ian) [inline docs/gdevelop5/interface/games-dashboard/player-feedback.md:51]:
```suggestion
![Rate player feedback](Rate-feedback.png)
```

File: docs/gdevelop5/interface/games-dashboard/player-feedback.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+On the game listing screen you can configure:
+
+* **Make discoverable on gd.games:** players will be avaliable to find the game through the gd.games site
+* **Show feedback banner on gd.page:** the game will display a banner to collect player feedback through the site gd.games
+* **Ask for feedback on all builds:** All published versions of the game listed under the "Builds" tab will dislay the feedback banner to collect player feedback
+
+To learn how to publish to the gd.games platform and configure the privacy of the links [follow these instructions](https://wiki.gdevelop.io/gdevelop5/publishing/web/)
+
+## Receiving feedback
+Recieved player feedback is stored and visible under the "Feedback" tab of the desired published Game.
```
Comment (4ian) [inline docs/gdevelop5/interface/games-dashboard/player-feedback.md:43]:
```suggestion
Received player feedback is stored and visible under the "Feedback" tab of the desired published Game.
```

File: docs/gdevelop5/interface/games-dashboard/player-feedback.md [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+Player feedback can be turned on or off acording to the game's development stage.
+
+To be able to collect player feedback the game has to be available online. You can verify that the Game has been published to gd.games through the [Manage section](https://wiki.gdevelop.io/gdevelop5/interface/games-dashboard/):
+
+![Manage section](Manage-Games.png)
+
+On the game listing screen you can configure:
+
+* **Make discoverable on gd.games:** players will be avaliable to find the game through the gd.games site
+* **Show feedback banner on gd.page:** the game will display a banner to collect player feedback through the site gd.games
```
Comment (4ian) [inline docs/gdevelop5/interface/games-dashboard/player-feedback.md:37]: gd.games website (not gd.games site or site gd.games)

File: docs/gdevelop5/interface/games-dashboard/player-feedback.md [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+4. Report feedback (spam)
+5. Resolve / Close feedback 
+
+Keep in mind that the creators that rate the most amount of player feedback, will be listed under the Community's Top 5 "Games with rated comments".
+
+![Community section](Community-section.png)
+
+### FAQ
+
+1. **If I get a bad review from a player, can it affect my game's visibility?** -> No, bad reviews do not affect the visibiliy of your game. Player feedback is private and it was concieved for games who are still under development and require player feedback. It is in no standard to score
```
Comment (4ian) [inline docs/gdevelop5/interface/games-dashboard/player-feedback.md:67]: > It is in no standard to score 

What does this means?

Also: missing period at the end + typo on conceived

File: docs/gdevelop5/interface/games-dashboard/player-feedback.md [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+5. Resolve / Close feedback 
+
+Keep in mind that the creators that rate the most amount of player feedback, will be listed under the Community's Top 5 "Games with rated comments".
+
+![Community section](Community-section.png)
+
+### FAQ
+
+1. **If I get a bad review from a player, can it affect my game's visibility?** -> No, bad reviews do not affect the visibiliy of your game. Player feedback is private and it was concieved for games who are still under development and require player feedback. It is in no standard to score 
+2. **What happens after you report a spam review?** -> If a user gets flagged after a certain number of times, their profile will be blocked and they'll no longe be able to give feedback
```
Comment (4ian) [inline docs/gdevelop5/interface/games-dashboard/player-feedback.md:68]: longe => longer

File: docs/gdevelop5/interface/games-dashboard/player-feedback.md [hunk 11]
```diff
@@ -0,0 +1,75 @@
...
+
+![Manage section](Manage-Games.png)
+
+On the game listing screen you can configure:
+
+* **Make discoverable on gd.games:** players will be avaliable to find the game through the gd.games site
+* **Show feedback banner on gd.page:** the game will display a banner to collect player feedback through the site gd.games
+* **Ask for feedback on all builds:** All published versions of the game listed under the "Builds" tab will dislay the feedback banner to collect player feedback
+
+To learn how to publish to the gd.games platform and configure the privacy of the links [follow these instructions](https://wiki.gdevelop.io/gdevelop5/publishing/web/)
```
Comment (4ian) [inline docs/gdevelop5/interface/games-dashboard/player-feedback.md:40]: missing period at the end of the sentence

File: docs/gdevelop5/interface/games-dashboard/player-feedback.md (hunk 7 above)
Comment (4ian) [reply docs/gdevelop5/interface/games-dashboard/player-feedback.md:43]: No capital letter to Game.

**Review by 4ian — APPROVED** [review]

**Pull request merged**